src/index.html

```diff
@@ -11,8 +11,44 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/variables.scss"
+    />
```

Comment:
You should not link `.scss` files in your HTML. Instead, you should compile them to `.css` and link the compiled `.css` file.